x-pack/plugins/cases/public/components/confirm_delete_case/index.tsx

```diff
@@ -31,20 +31,18 @@ const ConfirmDeleteCaseModalComp: React.FC<ConfirmDeleteCaseModalProps> = ({
     <EuiConfirmModal
       buttonColor="danger"
       cancelButtonText={i18n.CANCEL}
-      confirmButtonText={isPlural ? i18n.DELETE_CASES : i18n.DELETE_CASE}
+      confirmButtonText={i18n.DELETE_CASE(caseQuantity)}
       data-test-subj="confirm-delete-case-modal"
       defaultFocusedButton="confirm"
       onCancel={onCancel}
       onConfirm={onConfirm}
       title={
-        isPlural
-          ? i18n.DELETE_SELECTED_CASES
-          : caseTitle == null
+        caseTitle == null
```

Comment:
Is it a valid scenario? Why a case would have a 'null' title?

Reply:
I think this check is because `caseTitle` is optional.

Reply:
I understood why the type is nullable. Delete cases modal is always rendered. Even when it isn't displayed. I am changing `caseTitle` default value from `undefined` to `""`.